lib/registration.pm

```diff
@@ -464,7 +464,7 @@ sub process_scc_register_addons {
         wait_still_screen 2;
         # Process addons licenses
         accept_addons_license @scc_addons;
-        if (get_var('SCC_ADDONS') =~ /phub/ && check_screen('import-untrusted-gpg-key')) {
+        while (check_screen('import-untrusted-gpg-key', 60)) {
```

Comment:
is there any side effect remove *get_var('SCC_ADDONS') =~ /phub/* ?

Reply:
No, I have verified not impact other case:
https://openqa.nue.suse.com/tests/4052058